src/components/course/routes/ExternalCourseEnrollment.jsx

```diff
@@ -128,7 +128,7 @@ const ExternalCourseEnrollment = () => {
                     )}
                 </p>
               )}
-              <CourseSummaryCard courseMetadata={minimalCourseMetadata} />
+              <CourseSummaryCard />
```

Comment:
This component already used useMinimalCourseMetadata under the hood, so removing the prop here is a no-op.